src/anbox/network/message_processor.h

```diff
@@ -20,13 +20,15 @@
 
 #include <cstdint>
 #include <vector>
+#include "anbox/common/small_vector.h"
 
 namespace anbox {
 namespace network {
 class MessageProcessor {
  public:
   virtual ~MessageProcessor() {}
-  virtual bool process_data(const std::vector<std::uint8_t> &data) = 0;
+  virtual bool process_data(const std::vector<std::uint8_t> &) { return false; }
+  virtual bool process_data(anbox::common::SmallFixedVector<char, 512> &&) { return false; }
```

Comment:
I would like to use a 

`use MessageBuffer = SmallFixedVector<chat, 512>;`

here

Reply:
Yeah, will add that